### PR TITLE
PPU LLVM minor cleanup and minor fixes

### PR DESCRIPTION
**Pull request by Nekotekina**

### CODE REVIEW

File: rpcs3/Emu/Memory/vm.cpp [hunk 1]
```diff
@@ -80,15 +80,25 @@ namespace vm
...
 		u64 old = UINT64_MAX;
 		const auto cpu = get_current_cpu_thread();
 
 		while (true)
 		{
-			const auto [ok, rtime] = try_reservation_update(addr, size, lsb);
-			if (ok || old / 128 < rtime / 128)
+			const auto [ok, rtime] = try_reservation_update(addr);
+
+			if (ok || (old & -128) < (rtime & -128))
```
Comment (elad335) [inline rpcs3/Emu/Memory/vm.cpp:101]: why this change? the compiler already emitted the same code, it just looks works.

Comment (Nekotekina) [reply rpcs3/Emu/Memory/vm.cpp:101]: Well, we use this style everywhere else so kinda for consistency.
